db/migrate/20210330205846_move_family_members_to_clients.rb

```diff
@@ -0,0 +1,6 @@
+class MoveFamilyMembersToClients < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :short_intakes, :family_members, :jsonb
+    add_column :clients, :family_members, :jsonb, default: {}, using: 'family_members::jsonb'
```

Comment:
@vagarioustoast -- this would have lost all family_members data! I didn't _move_ the data, only the _column_....